docker-compose-local.yml

```
@@ -35,19 +35,18 @@ services:
       - theater_network
 
   mailhog:
-    restart: always
     build:
       context: .
       dockerfile: ./docker/mailhog/Dockerfile
     container_name: mailhog_theater
-    command: ["/bin/bash", "-c", "/commands/setup_mailhog_auth.sh && ~/go/bin/MailHog"]
+    command: [ "/bin/bash", "-c", "/commands/setup_mailhog_auth.sh && /usr/local/bin/mailhog -auth-file=/mailhog.auth" ]
```

MailHog command is provided as a shell string inside the array. Ensure the referenced script `/commands/setup_mailhog_auth.sh` exists inside the built image and is executable. If that script fails the container will keep restarting (see restart policy below) and the reason may be hidden. Consider logging or failing loudly from the script so problems are obvious during development.